src/main/java/org/apache/xml/security/stax/ext/XMLSecurityConstants.java

```diff
@@ -43,7 +43,8 @@ public class XMLSecurityConstants {
     public static final XMLOutputFactory xmlOutputFactory;
     public static final XMLOutputFactory xmlOutputFactoryNonRepairingNs;
 
-    private static final SecureRandom SECURE_RANDOM;
+    private static SecureRandom SECURE_RANDOM;
```

Comment:
Should this also be `volatile`?

Reply:
+1, thanks, I thought i made it volatile. Sorry.